README.md

```diff
@@ -9,11 +9,6 @@ The attempt here is to make a simple to use PHP SDK for @volcano.
 ## Example
 
 ```php
-require_once 'src/Service.php';
-require_once 'src/Sellers/Seller.php';
-
-$seller = new \VolcanoSDK\Sellers\Seller;
-
 \VolcanoSDK\Service::baseUrl('https://volcano/api');
```

Comment:
You'll need to change this line in the readme as well. There's also a test script which needs to be adjusted.

Otherwise looks good :+1: